#include "scene_server_player.pb.h"
#include "src/game_logic/thread_local/thread_local_storage.h"
#include "src/network/message_system.h"
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
void ServerPlayerSceneService::Gs2ControllerLeaveSceneAsyncSavePlayerComplete(entt::entity player,
	const ::Gs2ControllerLeaveSceneAsyncSavePlayerCompleteRequest* request,
	::google::protobuf::Empty* response)
{
///<<< BEGIN WRITING YOUR CODE
///<<< END WRITING YOUR CODE
}
